distribution/lib/Standard/Visualization/0.0.0-dev/src/Main.enso

```diff
@@ -9,4 +9,6 @@ export project.Helpers
 export project.Id.Id
 
 export project.Preprocessor
+export project.Table
```

Comment:
@kazcw @farmaazon I was unable to reproduce the problem with the `Table` visualization, only the `Warnings` one.

Reply:
Indeed, the Table visualization seems to work on develop right now.